package-readmes/gcp.md

```diff
@@ -0,0 +1,265 @@
+
+To use the functions below in your code, import them with the following syntax:
+
+```js 
+import { <function name> } from '@velo/gcp-cloud-trace-integration-backend';
+```
+
+* **traceRoot()**
+
+    Creates a root span (top level span) and traces it. The root span can include child spans using the `traceChild()` or `traceModule()` functions. 
+
+    Syntax:
+    
+    ```js
+    export async function traceRoot<T>(description: string, callback:() => T | Promise<T>): Promise<T>
+    ```
+
+    Parameters:
+    * **`description`**: The name of the root span as it appears in GCP Cloud Trace.
+    * **`callback`**: A callback function to trace. 
+
+    Returns:
+    
+    A promise that resolves to the callback function value when the callback function is successful.
+    
+    Example:  
+    
+	
+    ```js
+    export async function someWebModule(param1, param2) {
+      return traceRoot(<operation name>, async (span) => {
+        //... the function body
+      })
+    }
+    ```
+
+* **traceModule()**
+
+    Implements a module with a proxy for the specified Wix module object, and traces the APIs in that object. Note that this is the recommended way of working with Wix APIs. 
+
+    Syntax:
+
+    ```js
+    export function traceModule<T>(obj: T, name: string): T
+    ```
+    
+    Parameters:
+    * **`obj`**: Wix module object. 
+    * **`name`**: Wix module name. 
+
+    Returns:
+    
+    The Wix module object that was traced.
+    
+    Example:
+    
+    
+    ```js
+    const wixData = traceModule(require('wix-data'), 'wix-data');
+    const wixStoresBackend = traceModule(require('wix-stores-backend'), 'wix-stores-backend');
+    ```
+
+* **traceChild()**
+
+    Creates a child span and traces the callback with the span. Child spans can be nested. Note that `traceChild()` is the recommended way of working with fetch or getJSON APIs. 
+
+    Syntax:
+
+    ```js
+    export async function traceChild<T>(description: string, callback: () => T | Promise<T>): Promise<T>
+    ```
+
+    Parameters:
+    * **`description`**: The name of the child span as it appears in GCP Cloud Trace.
+    * **`callback`**: A callback function to trace. 
+
+    Returns:
+
+    A promise that resolves to the callback function value when the callback function is successful.
+    
+    Example:
+    
+    
+    ```js
+    export async function someWebModule(param1, param2) {
+      return traceRoot(<operation name>, async (span) => {
+        //... the function body
+        let res = await traceChild(<name of the fetch operation>, async (span) => {
+          return getJSON('someUrl');
+        })
+        //... the function body continues
+      })
+    }
+    ```
+
+* **currentSpan()**
+
+    Gets the current instance of [`Span`](https://open-telemetry.github.io/opentelemetry-js-api/interfaces/span.html). You can add events and attributes to the current span.
```

Comment:
Same link as above, led to 404 for me - https://open-telemetry.github.io/opentelemetry-js-api/interfaces/span.html